data/Makefile.am

```diff
@@ -39,8 +39,34 @@ install-sample-data:
 	rm fedora-20-icons.tar.gz; \
 	cd -
 
+com.endlessm.SystemExternalApps.rules: com.endlessm.SystemExternalApps.rules.in
```

Comment:
I chose to give a _com.endlessm_ namespace because without our plugin (and currently the plugin is not upstream) users wouldn't be able to use external apps and thus wouldn't need these extra rules (which also refer to a user that only our system has). I am okay with renaming it to org.gnome.Software if it's appropriate but I don't think it belongs at all to to org.freedesktop.Flatpak as I commented on the Phab task.